api/v1beta1/mariadbdatabase_funcs.go

```
@@ -228,6 +233,12 @@ func (d *Database) CreateOrPatchAll(
 		return nil
 	})
 
+	// set the database hostname on the db instance
```

i did this carelessly ignoring err for the above mariadbdatabase operation, need to fix this